src/utils.js

```diff
@@ -6,12 +6,17 @@ import fsExtra from 'fs-extra';
 import ApifyClient from 'apify-client';
 import psTree from 'ps-tree';
 import pidusage from 'pidusage';
+import requestPromise from 'request-promise';
 import _ from 'underscore';
+import XRegExp from 'xregexp';
 import { delayPromise } from 'apify-shared/utilities';
 import { ENV_VARS } from './constants';
 
 export const PID_USAGE_NOT_FOUND_ERROR = 'No maching pid found';
 
+export const URL_NO_COMMAS_REGEX = XRegExp('https?://(www\\.)?[-\\p{L}0-9@:%._\\+~#=]{2,256}\\.[a-z]{2,6}\\b([-\\p{L}0-9@:%_\\+.~#?&//=\\(\\)]*)', 'gi'); // eslint-disable-line
```

Comment:
We want these regexes to appear in documentation and to be included in `Apify.utils` so the need to be documented with `  * @memberof utils` and included in `publicUtils` below